src/store/index.ts

```diff
@@ -222,13 +225,13 @@ export interface IStore {
      */
     deleteAllData(): Promise<void>;
 
-    getOutOfBandMembers(roomId: string): Promise<MatrixEvent[] | null>;
+    getOutOfBandMembers(roomId: string): Promise<object[] | null>;
 
-    setOutOfBandMembers(roomId: string, membershipEvents: MatrixEvent[]): Promise<void>;
+    setOutOfBandMembers(roomId: string, membershipEvents: object[]): Promise<void>;
```

Comment:
These are surprising. What are they actually? (we shouldn't be using `object` - it's as good as `any`)

Reply:
`IEvent` I think, will confirm